llama-index-integrations/llms/llama-index-llms-dashscope/tests/test_dashscope.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from llama_index.core.base.llms.types import (
+    CompletionResponse,
+    ChatMessage,
+    ChatResponse,
+)
+from llama_index.llms.dashscope.base import DashScope
+
+
+@pytest.fixture()
+def dashscope_llm():
+    return DashScope(api_key="test")
+
+
+mock_response = {
+    "status_code": 200,
+    "request_id": "4438deec-2d21-9b9c-b405-a47459fd8f75",
+    "code": "",
+    "message": "",
+    "output": {
+        "choices": [
+            {
+                "finish_reason": "stop",
+                "message": {"role": "assistant", "content": "hi, there!"},
+            }
+        ]
+    },
+    "usage": {"total_tokens": 161, "output_tokens": 91, "input_tokens": 70},
+}
+
+
+def test_dashscope_complete(dashscope_llm, mocker):
```

Comment:
I think mocker should go in the fixture definition?
https://docs.pytest.org/en/stable/how-to/monkeypatch.html

Reply:
Ok, I will update my code